aware-core/src/main/java/com/aware/ui/esms/ESM_Question.java

```diff
@@ -357,6 +380,24 @@ private void dismissESM() {
         if (esm_dialog != null) esm_dialog.dismiss();
     }
 
+    /**
+     * When one of the ESM's has timed out, the entire queue gets removed.
+     */
+    public void removeESM(Context context) {
```

Comment:
This function name may lead the user to think it only removes one ESM. Rename to timeoutQueue